CMFA_ESR.adoc

```diff
@@ -103,78 +96,46 @@ The configuration process for CMFA is likely to encompass multiple steps, coveri
 == Use Case(s)
 CMFA is used primarily for continuous authentication of a user for computers such as smartphones, where the confidence rating is used to determine the locked state of the computer. 
 
-This confidence rating can also be provided to external services such as a PC login at the office, building or room entrance control or ATMs. Those external services may communicate CMFA to the computer so that the external services can request additional user data, such as specific sensor information, to perform secondary CMFA analysis before granting the access to a user.
+In addition to being used by the computer for determining the locked state, the confidence rating can also be provided other systems, such as applications installed on the computer or to external services such as a PC login at the office, building or room entrance control or ATMs. Those external services may communicate CMFA related information to the computer so that the external services can request additional user data, such as specific sensor information, to perform secondary CMFA analysis before granting the access to a user.
```

Comment:
Missing "to"

```suggestion
In addition to being used by the computer for determining the locked state, the confidence rating can also be provided to other systems, such as applications installed on the computer or to external services such as a PC login at the office, building or room entrance control or ATMs. Those external services may communicate CMFA related information to the computer so that the external services can request additional user data, such as specific sensor information, to perform secondary CMFA analysis before granting the access to a user.
```